src/lib.rs

```diff
@@ -88,7 +89,7 @@ mod format {
 
     /// Whether the pattern ends in the character `Z`.
     pub(crate) fn is_zulu(pattern: &str) -> bool {
-        pattern == YYYYMMDDHHMMSS_HYPHENATED_ZULU
+        pattern == YYYYMMDDHHMMSS_HYPHENATED_ZULU || pattern == YYYYMMDDHHMMSS_T_SEP_HYPHENATED_ZULU
```

Comment:
To match the comment you can simplify it to:
```suggestion
        pattern.ends_with('Z')
```